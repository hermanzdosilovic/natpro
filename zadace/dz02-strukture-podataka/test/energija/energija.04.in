500
ADD 480 86
ADD 203 15
ADD 329 92
ADD 423 19
ADD 201 40
ADD 356 19
ADD 358 22
ADD 51 45
ADD 369 83
ADD 477 91
ADD 79 12
ADD 428 61
ADD 132 89
ADD 279 76
ADD 474 73
ADD 159 29
ADD 11 38
ADD 415 58
ADD 280 36
ADD 295 48
ADD 340 69
ADD 182 80
ADD 269 92
ADD 81 4
ADD 114 76
ADD 38 86
DEL 182
QUERY 326
DEL 81
QUERY 1202
DEL 79
QUERY 364
DEL 269
QUERY 1039
DEL 415
QUERY 1492
DEL 295
QUERY 562
DEL 329
QUERY 1972
DEL 279
QUERY 728
DEL 474
QUERY 1645
DEL 203
QUERY 1265
DEL 477
QUERY 1077
DEL 159
QUERY 1406
DEL 132
QUERY 249
QUERY 247
QUERY 1075
ADD 409 83
ADD 22 27
ADD 380 57
QUERY 543
ADD 211 76
ADD 450 46
QUERY 347
QUERY 1677
ADD 156 95
QUERY 299
QUERY 1499
ADD 28 18
QUERY 635
QUERY 464
ADD 376 27
QUERY 768
ADD 249 70
QUERY 49
ADD 302 37
ADD 320 67
ADD 425 58
QUERY 962
QUERY 1207
ADD 82 83
QUERY 847
QUERY 149
QUERY 447
ADD 230 85
QUERY 1205
QUERY 1983
QUERY 1612
QUERY 212
ADD 467 93
QUERY 254
QUERY 798
ADD 144 83
QUERY 1626
QUERY 463
ADD 436 46
ADD 166 52
QUERY 1344
QUERY 240
QUERY 1653
ADD 225 87
QUERY 614
QUERY 1413
EXIT
