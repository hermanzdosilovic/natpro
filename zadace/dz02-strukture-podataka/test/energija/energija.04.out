11
11
11
11
11
11
38
11
11
11
11
11
11
11
11
22
22
22
22
22
28
22
28
11
28
28
28
11
22
28
28
28
11
22
28
28
22
28
11
28
28
28
